.gitignore

```diff
@@ -159,6 +159,14 @@ ajax/libs/Hyphenator/HyphenatorBeta12/
 ajax/libs/Hyphenator/ersion_*
 ajax/libs/gemma/0.1.0/
 ajax/libs/gemma/0.4.0/
+ajax/libs/teadao/0.1.0/
+ajax/libs/teadao/0.2.0/
+ajax/libs/teadao/0.2.1/
+ajax/libs/teadao/0.2.4/
+ajax/libs/teadao/0.2.5/
+ajax/libs/teadao/0.4.0/
+ajax/libs/teadao/0.5.0/
+
```

Comment:
@kennynaoh The author said that we can just start from 0.6.0. I think we can ignore `0.1.* ~ 0.5.*`. How do you think?